tests/scripts/delete_oidc_identities

```diff
@@ -6,3 +6,5 @@ lxd sql global "
     FROM identities
     WHERE name IN ('foo', 'bar');
 "
+
+sudo lxc config set user.show_permissions=false
```

Comment:
This is an unexpected side effect. When running the tests locally, this can leave the configuration in a different state as it was before. Maybe setting this config option shouldn't be part of the create and delete script? We can add it to the pr and coverage workflow instead, wdyt?

Reply:
Sure, although to run these tests locally, we'd have to make sure `user.show_permissions` is set to `true`. This is the primary reason why I included the commands in the scripts. Should we then maybe just include this in the repository setup steps in HACKING.md?

Reply:
We can also have those commands included in the `package.json` scripts, although that would make the scripts very long and verbose.

Reply:
I think in package.json we'd run into the same issue: Running the tests would mess with the LXD configuration. I'd be fine adding them to the HACKING.md file.

Reply:
okay sounds good, will add that :+1: